tests/test_question_templates/question_inputs/q14_string-input/q14_string-input.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Code Output
+topic: Template
+author: Gavin Kendal-Freedman
+source: original
+template_version: 1.0
+attribution: standard
+partialCredit: true
+singleVariant: false
+showCorrectAnswer: false
+outcomes:
+- 6.1.1.0
+- 6.1.1.1
+difficulty:
+- undefined
+randomization:
+- undefined
+taxonomy:
+- undefined
+span:
+- undefined
+length:
+- undefined
+tags:
+- unknown
+assets:
+server:
+    imports: |
+        import random; random.seed(111) 
+        import problem_bank_helpers as pbh
+    generate: |
+        data2 = pbh.create_data2()
+
+        a = random.randint(2, 4)
+        stringname = "love"
+
+        data2["params"]["vars"]["title"] = "Code Output"
+        data2["params"]["vars"]["a"] = a
+        data2["params"]["vars"]["stringname"] = stringname
+        data2["correct_answers"]["ans1"] = a * stringname
+
+        # Update the data object with a new dict
+        data.update(data2)
+part1:
+  type: string-input
+  gradingMethod: Manual
```

Comment:
This can also be removed once the above has been addressed (since it'll be auto graded)